src/fixhorizon.jl

```diff
@@ -135,37 +149,54 @@ POMDPs.reward(w::FHWrapper, ss::Tuple{<:Any,Int}, a) = reward(w.m, first(ss), a)
 POMDPs.actions(w::FHWrapper) = actions(w.m)
 POMDPs.actionindex(w::FHWrapper, a) = actionindex(w.m, a)
 POMDPs.discount(w::FHWrapper) = discount(w.m)
-POMDPModelTools.ordered_actions(w::FHWrapper) = ordered_actions(w.m)
 # TODO: convert_a
 
 #################################
 # distribution with a fixed stage
 #################################
+"""
+    InStageDistribution{D}
 
-# TODO: Define access functions for InStageDistribution - to access with method instead of .d or .stage
+Wrap given distribution with a given stage
+"""
 struct InStageDistribution{D}
     d::D
     stage::Int
 end
 
-function BeliefUpdaters.DiscreteBelief(pomdp, b::InStageDistribution; check::Bool=true)
-    return DiscreteBelief(pomdp, b.d; check)
+"""
+    distrib(d::InStageDistribution{D})::D
+
+Return distrubution wrapped in InStageDistribution without stage
+"""
+function distrib(d::InStageDistribution)
```

Comment:
Abbreviation is generally discouraged for things like this in Julia: https://docs.julialang.org/en/v1/manual/style-guide/#Use-naming-conventions-consistent-with-Julia-base/ (see the third bullet)